Python/Product/EnvironmentsList/PipExtension.xaml.cs

```diff
@@ -201,6 +203,8 @@ PipExtensionProvider provider
             _provider.IsPipInstalledChanged += PipExtensionProvider_IsPipInstalledChanged;
             _provider.InstalledPackagesChanged += PipExtensionProvider_InstalledPackagesChanged;
 
+            IsPipInstalled = _provider.IsPipInstalled ?? true;
```

Comment:
Null means package manager is null, why is pip considered installed in that case?